FiveCalls/FiveCalls/IssueDetailViewController.swift

```diff
@@ -174,7 +175,14 @@ extension IssueDetailViewController : UITableViewDataSource {
             
         case IssueHeaderRows.description.rawValue:
             let cell = tableView.dequeueReusableCell(withIdentifier: R.reuseIdentifier.descriptionCell, for: indexPath)!
-            cell.issueLabel.text = issue.reason
+
+            let markdown = Down.init(markdownString: issue.reason)
```

Comment:
`Down.init` should just be `Down` ya?

Reply:
I just like the `Down in it` usage. ;) 

But yeah bad habits, will fix in my second pass tonight. I'll try to hook up the UITextView so we can handle links as well.